py3status/modules/xrandr.py

```diff
@@ -333,7 +326,7 @@ def _refresh_py3status(self):
         """
         Send a SIGUSR1 signal to py3status to force a bar refresh.
         """
-        call(shlex.split('killall -s USR1 py3status'))
+        self.py3.command_run('killall -s USR1 py3status')
```

Comment:
This is horrible we should make it easier to refresh all the modules.

Anyhow we keep for now and find a nice solution soon